test/sender.test.js

```diff
@@ -402,117 +536,108 @@ describe('Sender message builder test suite (anything not covered in client inte
 
     it('ignores unfinished rows when preparing a buffer for send', function () {
         const sender = new Sender({bufferSize: 1024});
-        sender.table("tableName")
-            .symbol("name", "value")
-            .at("1234567890");
-        sender.table("tableName")
-            .symbol("name", "value2");
+        sender.table('tableName')
+            .symbol('name', 'value')
+            .at(1234567890n, 'ns');
+        sender.table('tableName')
+            .symbol('name', 'value2');
         expect(
             sender.toBufferView(sender.endOfLastRow).toString()
-        ).toBe("tableName,name=value 1234567890\n");
+        ).toBe('tableName,name=value 1234567890\n');
     });
 
     it('throws exception if a float is passed as integer field', function () {
         const sender = new Sender({bufferSize: 1024});
         expect(
-            () => sender.table("tableName")
-                .intColumn("intField", 123.222)
-        ).toThrow("Value must be an integer, received 123.222");
+            () => sender.table('tableName')
+                .intColumn('intField', 123.222)
+        ).toThrow('Value must be an integer, received 123.222');
     });
 
     it('throws exception if a float is passed as timestamp field', function () {
         const sender = new Sender({bufferSize: 1024});
         expect(
-            () => sender.table("tableName")
-                .timestampColumn("intField", 123.222)
-        ).toThrow("Value must be an integer or BigInt, received 123.222");
+            () => sender.table('tableName')
+                .timestampColumn('intField', 123.222)
+        ).toThrow('Value must be an integer or BigInt, received 123.222');
     });
 
     it('throws exception if designated timestamp is not a string or bigint', function () {
         const sender = new Sender({bufferSize: 1024});
         expect(
-            () => sender.table("tableName")
-                .symbol("name", "value")
-                .at(23232322323)
-        ).toThrow("The designated timestamp must be of type string or BigInt, received number");
-    });
-
-    it('throws exception if designated timestamp is an empty string', function () {
-        const sender = new Sender({bufferSize: 1024});
-        expect(
-            () => sender.table("tableName")
-                .symbol("name", "value")
-                .at("")
-        ).toThrow("Empty string is not allowed as designated timestamp");
+            () => sender.table('tableName')
+                .symbol('name', 'value')
+                .at(23232322323.05)
+        ).toThrow('Designated timestamp must be an integer or BigInt, received 23232322323.05');
     });
 
     it('throws exception if designated timestamp is invalid', function () {
```

Comment:
this test now looks the same as the previous one.
maybe we could change this to pass a string value.
```
.at('invalid_dts')
```

Reply:
Good catch. Fixed in 0a4380e11d58161e405e6d14fb50b32d4f75c1ff